lib/brook/storage/redis/migration.ex

```diff
@@ -30,6 +30,7 @@ defmodule Brook.Storage.Redis.Migration do
     old_events =
       Redix.command!(redix, ["LRANGE", events_key, 0, -1])
       |> Enum.map(&:erlang.binary_to_term/1)
+      |> Enum.map(fn event -> Map.update(event, :author, "migrated_default", &Supervisor.Default.init/1) end)
```

Comment:
It's the perfect identity function, really.

Reply:
The author reviewing his own pr regarding an identity function regarding a default author.
Yeah.... that's meta.